computer-use-demo/README.md

```diff
@@ -119,6 +109,18 @@ This example shows how to use the Google Cloud Application Default Credentials t
 
 You can also set `GOOGLE_APPLICATION_CREDENTIALS` to use an arbitrary credential file, see the [Google Cloud Authentication documentation](https://cloud.google.com/docs/authentication/application-default-credentials#GAC) for more details.
 
+### Accessing the demo app
```

Comment:
can you add a link to this section from the place that you moved it from?

Reply:
Done!